examples/evaluate_multiple_isolated_robots/main.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
```

Comment:
Ok I can remove that if desired, it sounds like it doesn't work in windows, just linux and mac.  The advantage though is that you can run the example scripts by just typing `./main.py` instead of `python ./main.py`. I personally like to have it for all my python scripts (and I make them executable with `chmod a+x` so the `./main.py` trick works). And of course you can always run the script with `python ./main.py` still.

https://realpython.com/python-shebang/

Reply:
Yeah I'm familiar with shebang and it would work for any sane developer. I'm a bit scared that students will simply run the script but not pay attention to their virtual environment and everything. Not sure if it's overcautious but I think it might be best to have students explicitly call python so they see exactly what they are doing. You'll be surprised about what simple things can lead to confused students haha

Reply:
then again maybe its good practise? I don't know

Reply:
Ok let's leave it out just to prevent the students breaking their head over what this line means

Reply:
Done ✅